lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -172,6 +161,53 @@ def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier: Secur
 
         reuse_identifier
       end
+
+      # Update a milestone for a repository
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] number The number of the milestone we want to fetch
+      # @param options [Hash] A customizable set of options.
+      # @option options [String] :title A unique title.
+      # @option options [String] :state
+      # @option options [String] :description A meaningful description
+      # @option options [Time] :due_on Set if the milestone has a due date
+      # @return [Milestone] A single milestone object
+      #
+      def self.update_milestone(repository:, number:, options: {})
+        client.update_milestone(repository, number, options)
+      end
```

Comment:
I like the detailed documentation here, and nice use of the `@option` YARD tag as well, TIL 👍 
I also like the fact that this wrapper adds named parameters, making the call site clearer 👍 

💄  Given the list of options come from the documentation of the REST API, I wonder if it might be worth adding a `# @see https://docs.github.com/en/rest/issues/milestones#create-a-milestone` as well, for reference? 🤷